pkg/stencil/lockfile.go

```diff
@@ -81,3 +84,70 @@ func LoadLockfile(path string) (*Lockfile, error) {
 	err = yaml.NewDecoder(f).Decode(&lock)
 	return lock, err
 }
+
+// MergeMissingFilesFromOlderLockfile merges missing files from the older lockfile into the current one, for use with file.Once.
+func (lf *Lockfile) MergeMissingFilesFromOlderLockfile(older *Lockfile) {
+	for _, f := range older.Files {
+		if !slices.ContainsFunc(lf.Files, func(fe *LockfileFileEntry) bool {
+			return fe.Name == f.Name
+		}) {
```

Comment:
Pretty bad perf.. would be better with a hashmap. Not going to block over this though

Reply:
yeah but it's a list of 100 things, so idgaf :)